Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/ClientApp/src/app/utilities/translations.ts

```diff
@@ -0,0 +1,105 @@
+export let Translations = {
+  Dashboard: {
+    Title: {
+      en: 'Welcome to the FOFA Content Portal!',
+      de: 'Willkommen im FOFA Content Portal!',
+    },
+    DemoInfo: {
+      en: 'More info about this demo',
+      de: 'Weitere infos zu dieser demo',
+    },
+    Content: {
+      en: `Our association is all about sharing knowledge, logistics and resources to promote organic farming - to both
+      farmers and the public. As a valued member of our organization, you can access carefully researched and approved
+      content for reuse via this Content Portal.`,
+      de: `In unserem Verein geht es darum, Wissen, Logistik und Ressourcen zu teilen, um den ökologischen Landbau zu 
+      fördern - für beide Landwirte und die Öffentlichkeit. Als geschätztes Mitglied unserer Organisation können Sie auf 
+      sorgfältig recherchierte und genehmigte Daten zugreifen Inhalte zur Wiederverwendung über dieses Inhaltsportal.`,
```

Comment:
I would change the sentence starting with Als geschätztes mitglied to: Als geschätztes Mitglied unserer Organisation können Sie über dieses Content Portal auf sorgfältig recherchierte und genehmigte Inhalte zur Wiederverwendung zugreifen.

Reply:
Done